sdk/formrecognizer/azure-ai-formrecognizer/tests/test_custom_forms.py

```diff
@@ -61,7 +64,8 @@ def test_custom_form_multipage_unlabeled(self, client, formrecognizer_multipage_
 
     @FormRecognizerPreparer()
     @FormTrainingClientPreparer()
-    def test_custom_form_labeled(self, client, formrecognizer_storage_container_sas_url_v2):
+    @recorded_by_proxy
+    def test_custom_form_labeled(self, client, formrecognizer_storage_container_sas_url_v2, **kwargs):
```

Comment:
are you getting an error if you don't add kwargs?

Reply:
Yes I am, I noticed that the tests that take in several params need it or else will raise an error.

Reply:
what is the error? wondering if this is a bug we need to open.

Reply:
I thought this was expected behavior because of what's said here: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/test_proxy_migration_guide.md#record-test-variables

The error basically says that a `variables` kwarg is being passed in to these tests and it's not specified. That's why I added the **kwargs to these tests. Let me know if you think this is a bug.

Reply:
I'm curious about this because when I converted over TA, my tests also took params but I didn't need to include kwargs. I read that section of the migration guide as you need only accept `variables` or `kwargs` if you want to use recorded test variables (which we don't do in FR yet). I'll look into this

Reply:
Okay, seems to work in live/playback with or without kwargs. I don't think it hurts to keep it, though.

Reply:
Yeah I've run all the tests on my local machine both live and in playback and (after adding the kwargs) all the tests ran fine